src/main/java/genie/Storage.java

```diff
@@ -15,14 +15,14 @@
 public class Storage {
     private static final String PATH = "./data/duke.txt";
     private static FileWriter fw;
```

Comment:
Perhaps we could use a longer variable name here, since the scope in which this FileWriter is used is quite large?